certified-connectors/PKIsigning/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2349 @@
+        },
+        "reasons": {
+          "type": "string",
+          "description": "An extra information field."
+        },
+        "signerNote": {
+          "type": "string",
+          "description": "An extra information field that will be shown to the signer."
+        },
+        "recipientNote": {
+          "type": "string",
+          "description": "An extra information field that will be shown to the recipient."
+        },
+        "accorderNote": {
+          "type": "string",
+          "description": "An extra information field that will be shown to the accorder."
+        },
+        "expires": {
+          "type": "string",
+          "description": "The date and time when the dossier will expire.",
+          "format": "date-time"
+        },
+        "emailActivity": {
+          "type": "array",
+          "description": "A list of the emails that have been send.",
+          "items": {
+            "$ref": "#/definitions/SignerActivity"
+          }
+        },
+        "myDossier": {
+          "type": "boolean",
+          "description": "A value indicating if the account associated with the request owns the dossier."
+        },
+        "taxPaymentStatus": {
+          "type": "string",
+          "description": "An extra information field that indicates the status of the tax payment."
+        },
+        "workgroup": {
+          "type": "string",
+          "description": "this is the description of the workgroup",
+          "format": "guid",
+          "x-ms-visibility": "advanced"
+        },
+        "clearancelevel": {
+          "type": "integer",
+          "description": "A dossier is only visible to other users with the same or higher clearance level with a workgroup",
+          "format": "int32",
+          "x-ms-visibility": "advanced",
+          "default": 1
+        }
+      }
+    },
+    "DocumentModel": {
+      "type": "object",
+      "required": [
+        "id",
+        "documentType",
+        "dossierIndex",
+        "documentstatus"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique ID of the dossier",
+          "format": "guid"
+        },
+        "documentName": {
+          "type": "string",
+          "description": "Name of the Dossier the document belongs to"
+        },
+        "documentType": {
+          "$ref": "#/definitions/DocumentTypes"
+        },
+        "dossierIndex": {
+          "type": "integer",
+          "description": "Position within the dossie",
+          "format": "int32"
+        },
+        "documentstatus": {
+          "$ref": "#/definitions/DocumentStatuses"
+        },
+        "actors": {
+          "type": "array",
+          "description": "Actors within the document",
+          "items": {
+            "$ref": "#/definitions/Signer"
+          }
+        }
+      }
+    },
+    "DocumentTypes": {
+      "type": "string",
+      "description": "",
```

Comment:
@pkisigning Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
all empty descriptions now either have a value, or have been removed.